packages/xod-client/src/user/containers/AccountPane.jsx

```diff
@@ -0,0 +1,88 @@
+import R from 'ramda';
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+
+import { foldMaybe } from 'xod-func-tools';
+
+import Button from '../../core/components/Button';
+
+import * as Actions from '../actions';
+import * as Selectors from '../selectors';
+
+import AuthForm from '../components/AuthForm';
+
+const AccountPane = ({
+  user,
+  isVisible,
+  compilationsLeft,
+  actions,
+  isAuthorising,
+  authError,
+}) => {
+  if (!isVisible) return null;
+
+  return (
+    <div className="AccountPane">
+      <a
+        role="button"
+        tabIndex="0"
+        className="close-button"
+        onClick={actions.toggleAccountPane}
+      >
+        &times;
+      </a>
+
+      {foldMaybe(
+        <div className="username">Guest XODer</div>,
```

Comment:
If “Xoder” is a legal word it should be cased like a regular word: “Guest Xoder”